src/yandex_cloud_ml_sdk/_files/domain.py

```diff
@@ -18,6 +18,9 @@
 
 
 class BaseFiles(BaseDomain, Generic[FileTypeT]):
+    """Base class for file operations in Yandex Cloud.
+    Provides methods to upload files either as bytes or from the path.
```

Comment:
Also `get` and `list` methods, which is allows you to work with remote files object you created earlier)))